curiouslearning/FeedTheMonsterJS
# fix: Cr 176 content changes for Tigragna

# Changes
- lang/tigragna/ftm_tigragna.json

# How to test
- When I play the development APK and open Tigragna,
Then I should see the title as “ዓርክኻ መግቦ”.

Ref: [CR-176](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-176)


[CR-176]: https://curiouslearning.atlassian.net/browse/CR-176?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
